tableauserverclient/server/server.py

```diff
@@ -83,6 +84,7 @@ def __init__(self, server_address, use_server_version=False):
         self.data_acceleration_report = DataAccelerationReport(self)
         self.data_alerts = DataAlerts(self)
         self._namespace = Namespace()
+        self.flowruns = FlowRuns(self)
```

Comment:
I think it should be `flow_runs` to match others like `data_alerts`

Reply:
Changed to `flow_runs`.